src/Controller/Component/CrudComponent.php

```diff
@@ -218,9 +218,7 @@ public function execute($controllerAction = null, $args = [])
         $this->_action = $controllerAction ?: $this->_action;
 
         $action = $this->_action;
-        if (empty($args)) {
-            $args = $this->_request->params['pass'];
-        }
+        $args = (!empty($args)) ? [$args] : $this->_request->params['pass'];
```

Comment:
This isn't right, you should be using `[["key1","key2"]]` when calling the method not mucking up the args inside `execute()`. This change will break all existing actions which take multiple args.